docs/quickstart.mdx

```diff
@@ -88,13 +79,24 @@ Submit the form and TEST be on your way.
 
 This is a Sapphire tutorial and you're already a Solidity expert, so let's not
 bore you with explaining the gritty details of the contract.
-Start by pasting [Vigil.sol] into `contracts/Vigil.sol`.
+Start by pasting `Vigil.sol` into `contracts/Vigil.sol`.
+
+1. Create a new file called `Vigil.sol` under `contracts/`:
+2. Paste the following contract into it:
 
-While you're there, also place [run-vigil.ts] into `scripts/run-vigil.ts`.
-We'll need that later.
+<details>
+  <summary> Vigil.sol contract </summary>
 
-[Vigil.sol]: https://github.com/oasisprotocol/sapphire-paratime/blob/stable/clients/js/1.x/examples/hardhat/contracts/Vigil.sol
-[run-vigil.ts]: https://github.com/oasisprotocol/sapphire-paratime/blob/stable/clients/js/1.x/examples/hardhat/scripts/run-vigil.ts
+  ![code Solidity](../examples/hardhat/contracts/Vigil.sol)
+</details>
+
+3. Add deploy and test tasks to Hardhat config:
```

Comment:
I would try to avoid the "test" word in the tasks not to confuse newcomers that are not used to hardhat or dApps in general. Hardhat tasks are meant for contract and dApp management. Tests should live separately in the test/ folder.

```suggestion
3. Add deploy and check tasks to Hardhat config:
```